enhancements/installer/manifests-from-stdin.md

```diff
@@ -0,0 +1,114 @@
+---
+title: manifests-from-sdtin
+authors:
+  - "@oglok"
+  - "@copejon"
+reviewers:
+  - "@fzdarsky"
+approvers:
+  - "@crawford"
```

Comment:
Please add @staebler

Reply:
@dgoodwin too, I'm not sure if this would have value for Hive.

Reply:
Do you want me to add them as reviewers? or approvers?

Reply:
Approvers